### PR TITLE
Add com.github.Anuken.Mindustry

### PR DESCRIPTION
**Pull request by muelli**
https://mindustrygame.github.io/

https://github.com/Anuken/Mindustry/issues/1732

### CODE REVIEW

File: com.github.Anuken.Mindustry.appdata.xml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    structures from waves of enemies. Play with your friends in 
+    cross-platform multiplayer co-op games, or challenge them in 
+    team-based PvP matches.
+  </p>
+ </description>
+ <screenshots>
+  <screenshot type="default">
+    <image>https://mindustrygame.github.io/images/1.png</image>
+    <caption>Defend your base from waves of powerful enemies</caption>
+  </screenshot>
```
Comment (bilelmoussaoui) [inline com.github.Anuken.Mindustry.appdata.xml]: Needs release & oars tags

**Comment by nedrichards** [comment]
Upstream does not seem keen. In fact, somewhat against the concept.

File: com.github.Anuken.Mindustry.yml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        - install -D com.github.Anuken.Mindustry.png /app/share/icons/hicolor/512x512/apps/com.github.Anuken.Mindustry.png
+
+    sources:
+      - type: file
+        url: https://f-droid.org/repo/io.anuke.mindustry/en-US/icon_Eno3XvqCZUcHRm3eMjiUleAxgzLopPe6-hkI7BHx1lU=.png
+        dest-filename: com.github.Anuken.Mindustry.png
+        sha256: 127a375efa82654707466dde32389495e0318332e8a4f7bafa1908ec11f1d655
+        
+      - type: file
+        path: com.github.Anuken.Mindustry.appdata.xml
```
Comment (LeandroStanger) [inline com.github.Anuken.Mindustry.yml]: New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html

Comment (muelli) [reply com.github.Anuken.Mindustry.yml]: I've renamed the file to "metainfo". I hope that is sufficient.

File: com.github.Anuken.Mindustry.appdata.xml (hunk 1 above)
Comment (muelli) [reply com.github.Anuken.Mindustry.appdata.xml]: added

**Comment by LeandroStanger** [comment]
bot, build com.github.Anuken.Mindustry

File: com.github.Anuken.Mindustry.yml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+        sha256: 127a375efa82654707466dde32389495e0318332e8a4f7bafa1908ec11f1d655
+        
+      - type: file
+        path: com.github.Anuken.Mindustry.metainfo.xml
+
+      - type: file
+        path: com.github.Anuken.Mindustry.desktop
+
+      - type: file
+        path: mindustry.sh
```
Comment (barthalion) [inline com.github.Anuken.Mindustry.yml]: Build files as this file is missing.

**Comment by barthalion** [comment]
It would be nice to try to submit appdata/desktop file to upstream nonetheless, but given their stance, I'll the decision to you.

/merge

**Pull request closed without merging**